src/main.rs

```diff
@@ -10,37 +11,48 @@ struct Cli {
 
 #[derive(Subcommand)]
 enum Commands {
-    /// Adds a package to the project
+    /// Adds packages to the project
     Add {
```

Comment:
When I run `noir-libs add` without any package I got no message. Can we in that case print the same what `noir-libs add --help`?

Reply:
Fixed!